src/components/Footer/Footer.module.scss

```
@@ -0,0 +1,77 @@
+.background {
+  background-color: $footer-background;
+
+  width: 100%;
+  height: 16rem;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+
+  padding: 3.2rem 0 10.8rem;
+  padding-left: 11.1rem;
+  padding-right: 11.1rem;
+  margin: 0 auto;
+}
+
+.container {
+  color: $footer-font-color;
+  width: 120rem;
+  height: 2rem;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.copyright,
+.link {
+  font-family: Arial;
+  font-size: 16px;
```

```suggestion
  font-size: 1.6rem;
```

px이 몇군데 더 있네요 그것도 같이 수정 부탁드립니다